compiler-rt/lib/sanitizer_common/sanitizer_procmaps_bsd.cpp

```diff
@@ -30,17 +30,17 @@ namespace __sanitizer {
 
 #if SANITIZER_FREEBSD
 void GetMemoryProfile(fill_profile_f cb, uptr *stats) {
```

Comment:
stack size of stack frame size limit?
if the latter, maybe noinline on the function is better fix?

Reply:
if you meant adding `NOINLINE` it fixes nothing.

Reply:
would it be possible fixup title to avoid line break?

How bit this struct? Or what is error message?

I would prefer RAII with InternalScopedVector<kinfo_proc> of size 1

Reply:
> would it be possible fixup title to avoid line break?
> 
> How bit this struct? Or what is error message?

1088 :)